src/shared/api/generated/CBioPortalAPI.ts

```diff
@@ -6519,11 +6518,13 @@ export default class CBioPortalAPI {
      */
     createDataAccessTokenUsingPOST(parameters: {
         'allowRevocationOfOtherTokens' ? : boolean,
-        $queryParameters ? : any
+        $queryParameters ? : any,
```

Comment:
@angelicaochoa Did you edit this file manually? This is an auto-generated file, your changes will be overridden when someone runs `npm run updateAPI` command.

Reply:
@ersinciftci we auto-generated the file initially. I just made these changes manually since there weren't many

Reply:
@angelicaochoa I see, but this file is not meant to be edited manually, it's auto generated from `CBioPortalAPI-docs.json`, so the next time someone runs the generation script, your changes will be gone. And this is the client file for public API anyway, the internal API's file is `CBioPortalAPIInternal.ts`. But you are not supposed to edit them manually, you need to point the frontend to your local (assuming your local is running the changes in https://github.com/cBioPortal/cbioportal/pull/5225),  and run `npm run updateAPI`.

Reply:
@ersinciftci ah, okay. thanks 👍